app/src/main/java/com/example/qrchive/Fragments/CaptureFragment.java

```diff
@@ -130,7 +132,10 @@ public void onImageSaved(@NonNull ImageCapture.OutputFileResults outputFileResul
                                 outputFileResults.getSavedUri();
                         Toast.makeText(requireContext(), msg, Toast.LENGTH_LONG).show();
                         Log.d(TAG, msg);
-                        getParentFragmentManager().beginTransaction().replace(R.id.fragment_container,new ScanFragment(fbw),null).commit();
```

Comment:
This was an issue. The fragment wasn't correctly added to the stack, but now it is.